src/jobservice/logger/backend/std_logger_test.go

```diff
@@ -1,7 +1,6 @@
 package backend
 
 import "testing"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380